tasks/dyndns.yml

```diff
@@ -0,0 +1,41 @@
+---
+
+- name: DynDNS - remove DynDNS
+  delegate_to: localhost
+  community.general.xml:
+    path: "{{ local_config_path }}"
+    xpath: /opnsense/OPNsense/DynDNS{{ item }}
```

Comment:
I would leave this a task for `opn_unset`
But if you prefer to do it in the dyndns task, you should just expect a list of UUIDs in the VAR and then use
```yaml
xpath: "/opnsense/OPNsense/DynDNS/accounts/account[@uuid='{{ item }}']
```
to remove accounts